src/js/common/index.js

```diff
@@ -10,12 +10,12 @@ require('./utils/sticky-action-box.js');
 // New navigation menu
 require('../legacy/mega-menu.js');
 
-// Disable scrolling on input[type="number"] because it was changing the value unexpectedly
-// NOTE: This also just stops the user from scrolling when hovering over a number
-//  input that has focus. This isn't ideal, but until a solution is found to that,
-//  this is what we've got.
-document.addEventListener('mousewheel', (event) => {
+// Prevent some browsers from changing the value when scrolling while hovering
```

Comment:
Thanks @cvalarida, is the focus bit still true? I thought that's what the second boolean clause was ensuring before.